packages/nc-gui/components/cell/MultiSelect.vue

```diff
@@ -271,7 +271,7 @@ const onTagClick = (e: Event, onClose: Function) => {
     class="w-full"
     :bordered="false"
     clear-icon
-    show-search
+    :show-search="active || editable"
```

Comment:
by changing this, it would throw `onSearch should work with showSearch instead of use alone.` on console.